### PR TITLE
9.0 mgmtsystem probability

### PR DESCRIPTION
**Pull request by naousse**

### CODE REVIEW

**Review by max3903 — CHANGES_REQUESTED** [review]
@naousse Same comments as in #157

File: mgmtsystem_probability/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+{
+    "name": "Management System Probability",
+    "version": "9.0.1.0.0",
+    "author": "Savoir-faire Linux,Odoo Community Association (OCA)",
+    "website": "http://www.savoirfairelinux.com",
+    "license": "AGPL-3",
+    "category": "Management System",
+    "complexity": "normal",
```
Comment (dreispt) [inline mgmtsystem_probability/__openerp__.py]: Deprecated

File: mgmtsystem_probability/__init__.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
```
Comment (dreispt) [inline mgmtsystem_probability/__init__.py]: It would be nice to use the short license disclaimers

File: mgmtsystem_probability/data/mgmtsystem_probability.xml [hunk 3]
```diff
@@ -0,0 +1,63 @@
+<odoo>
+    <data>
```
Comment (dreispt) [inline mgmtsystem_probability/data/mgmtsystem_probability.xml]: Should we use `noupdate="1"`?

File: mgmtsystem_probability/model/mgmtsystem_probability.py [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+
+from openerp import fields, models, _
+
+
+class MgmtSystemProbability(models.Model):
+
```
Comment (dreispt) [inline mgmtsystem_probability/model/mgmtsystem_probability.py]: Remove this blank line

File: mgmtsystem_probability/tests/__init__.py [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+from . import test_create_probability
+
+checks = [
+    test_create_probability
+]
```
Comment (dreispt) [inline mgmtsystem_probability/tests/__init__.py]: Why is this needed? Might it be deprecated?

**Review by dreispt — COMMENTED** [review]
Small nitpicks, but OK otherwise

File: mgmtsystem_probability/README.rst [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+used across different modules (health and safety, information security).
+
+Installation
+============
+
+To install this module, you need to:
+
+ * Clone the project from github on your instance
+    * git clone https://github.com/OCA/management-system.git
+ * Start odoo with the project in the addons path
```
Comment (max3903) [inline mgmtsystem_probability/README.rst]: @naousse No need for content here.

File: mgmtsystem_probability/README.rst [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+
+No configuration needed
+
+Usage
+=====
+
+To use this module, you need to:
+
+ * go to Management System > Configuration > Probability
+
```
Comment (max3903) [inline mgmtsystem_probability/README.rst:21]: @naousse Please add the link to Runbot here.

File: mgmtsystem_probability/README.rst [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+ None
+
+Credits
+=======
+
+Contributors
+------------
+
+* Loïc Faure-Lacroix <loic.lacroix@savoirfairelinux.com>
```
Comment (max3903) [inline mgmtsystem_probability/README.rst:41]: @naousse Add yourself

File: mgmtsystem_probability/data/mgmtsystem_probability.xml (hunk 3 above)
Comment (max3903) [reply mgmtsystem_probability/data/mgmtsystem_probability.xml]: Yes!

**Review by max3903 — CHANGES_REQUESTED** [review]

**Review by max3903 — APPROVED** [review]
